python_modules/dagster/dagster/_core/execution/backfill.py

```diff
@@ -62,13 +62,15 @@ class BulkActionsFilter:
         tags (Optional[Dict[str, Union[str, List[str]]]]): A dictionary of tags to query by. All tags specified
             here must be present for a given bulk action to pass the filter.
         job_name (Optional[str]): Name of the job to query for. If blank, all job_names will be accepted.
+        backfill_ids (Optional[Sequence[str]]): A list of backfill_ids to filter by. If blank, all backfill_ids will be included
     """
 
     statuses: Optional[Sequence[BulkActionStatus]] = None
     created_before: Optional[datetime] = None
     created_after: Optional[datetime] = None
     tags: Optional[Mapping[str, Union[str, Sequence[str]]]] = None
     job_name: Optional[str] = None
+    backfill_ids: Optional[Sequence[str]] = None
```

Comment:
BulkActionsFilter isn't part of the public api, but i could still mark the new param as experimental. of the ones i've added recently, this one seems the most likely to be removed/changed

Reply:
it's not public, so I think we're fine here